stream1.map

```diff
@@ -0,0 +1,742 @@
+E0079.WAV sfx/towners/bmaid01.wav
```

Comment:
@john-tornblom, I heard from @AJenbo you did a spectral analysis to correlate these sounds. Sounds really cool. Do you have the source for this online somewhere, or care to share more about the method you used?

Cheers,
Robin

Reply:
Hey, what I did was very simple and crude, but I've added the python code to: 
   https://gist.github.com/john-tornblom/5bd680955e3b026833f78efcb622e3b7

The most basic thing to do is to look at the duration of the waves, which turned out to be way too simple for the diablo audio. Next, I looked at the average amplitude in the frequency domain, which was sufficient for this particular case.

If you wanna improve things even more, I can recommend the following blog-post:
  https://towardsdatascience.com/a-data-scientists-approach-to-visual-audio-comparison-fa15a5d3dcef

/John

Reply:
Is spectral analysis really fool proof though? I just aligned up the enum with the array strings on the PC release, was very easy and quick.

Reply:
I think your approach is pretty solid. Had I known about your approach a week ago I would have done thing differently :). What I did find out however is the presence of superfluous files on the pc cdrom (or so I believe). For example, the ps1 file *0295.vag is present in two places on the pc cdrom, Sfx/Rogue/Rogue50.wav and Sfx/Animals/Rogue50.wav. I assume the latter is a mistake :)

Reply:
Unless they considered humans animals at one point lol My script doesn't convert alot of the unused audio and only does the stuff that's used, the plan was to strip out all non-speech files for language packs since you dont want lower quality assets. The enum trick was pretty simple, though the PSX had some new sounds:
`(c)0058.VAG -> LS_FLAMWAVE (enum) -> "Sfx\\Misc\\Flamwave.wav"`

Reply:
Right, makes sense. Sure, you can add more stuff to the PC version, but one step at a time imho. I just wanna experience Diablo in Swedish with decent controls :)

Reply:
I get that, alot of people didn't like playing on the PSX. It's also a nice way to learn a new language, since we only learn English in my country ;P

Reply:
> Right, makes sense. Sure, you can add more stuff to the PC version, but one step at a time imho. I just wanna experience Diablo in Swedish with decent controls :)

Haha, awesome. John är du från sverige? :)

Reply:
> > Right, makes sense. Sure, you can add more stuff to the PC version, but one step at a time imho. I just wanna experience Diablo in Swedish with decent controls :)
> 
> Haha, awesome. John är du från sverige? :)

Yepp, born and raised, what gave it away? :smile:

Reply:
haha ;) What part of Sweden? I'm in Stockholm.

Reply:
Ah, cool :)

I've been living in Linköping for the past 12 years (studies brought me here), but I'm originally from Västerås.

Reply:
Ah, cool! One of my childhood friends studied in Linköping too!  So I've been there a lot, really lovely place. A lot of bikes :)

Once the Covid-19 situation cools down a bit, @AJenbo and I and intending to organize a fika-hackathon. John, you'd be most welcome to join us if you wish!

Cheers,
Robin

Reply:
Sounds fun! If I'm available, why not! You should know though, I've actually never really played Diablo 1 :smile:.  I did play Diablo 2 back in the day, but I intend to play the original in Swedish when/if its playable on modern PCs :wink:

Reply:
> Sounds fun! If I'm available, why not! You should know though, I've actually never really played Diablo 1 smile. I did play Diablo 2 back in the day, but I intend to play the original in Swedish when/if its playable on modern PCs wink

Haha, perfect motivation. And no worries at all. You are more than welcome. I've also played more Diablo 2 than 1, or at least Diablo 2 in multiplayer. Diablo 1 still has the best atmosphere and darkest feel in my opinion. Both are great games.

Reply:
> playable on modern PCs wink
Should be very playable :) Next release will have a few improvements to the rendering though if you want to wait for that.

Reply:
> to play the original in Swedish when/if its playable on modern PCs
Playing diablo for the first time on PC properly translated will be fun. First step, lean Swedish. Then Japanese :P